src/lib/storage/table.cpp

```diff
@@ -152,28 +164,13 @@ void Table::remove_chunk(ChunkID chunk_id) {
 
 void Table::append_chunk(const Segments& segments, std::shared_ptr<MvccData> mvcc_data,
                          const std::optional<PolymorphicAllocator<Chunk>>& alloc) {
-  Assert(_type != TableType::References || !mvcc_data,
-         "Setting explicit MvccData on a reference Table makes no sense. Reference Tables should use MvccData of "
-         "referenced Table, if any");
   Assert(_type != TableType::Data || static_cast<bool>(mvcc_data) == (_use_mvcc == UseMvcc::Yes),
          "Supply MvccData to data Tables iff MVCC is enabled");
```

Comment:
I know you didn't touch that, but since you are changing the file anyway: iff -> if

Reply:
@mrks maybe we shouldn't use "iff", but "IFF" to connotate we actually intended the second "f"?

Reply:
That would be another convention that people have to know, remember, and follow. While I would have thought that it should be commonly known, it was brought up so often by many different reviewers that I wonder if typing *if and only if* would save us some trouble.

Reply:
Might be me, a person who struggles to write a for loop without any errors, but I haven't heard of `iff` before. I was sure it was a typo and if there would be an `IFF` I would be sure that some idiot wanted to emphasize that he hates orthography.

Reply:
Didn't know that... TIL 😉

Reply:
Schweiger?

Reply:
"Today I Learned" - I also wanted to add my share of obscure abbreviations ;)